account_check_printing_report_base/data/report_paperformat_parameter.xml

```diff
@@ -6,4 +6,11 @@
                ref="account_check_printing_report_base.paperformat_check"/>
         <field name="name">--disable-smart-shrinking</field>
     </record>
+
+    <record id="paperformat_base_a4_parameter_disable_smart_shrinking"
+            model="report.paperformat.parameter">
+        <field name="paperformat_id"
+               ref="account_check_printing_report_base.paperformat_check_a4"/>
+        <field name="name">--disable-smart-shrinking</field>
```

Comment:
This is way the other dependency is mentioned in INSTALL, as this doesn't work without it.

Reply:
This file was not declared in manifest, so I delete it